README.md

```diff
@@ -46,24 +47,37 @@ Please read [bezier_library/doc/README.md](bezier_library/doc/README.md).
 Quick help
 ----------
 
-**Build**
+**Build using [wstool](http://wiki.ros.org/wstool) and [catkin_tools](http://catkin-tools.readthedocs.io/en/latest/)**
 
-Clone this repository into your catkin source folder and build the workspace:
 ```bash
-cd $(catkin_workspace)/src
-git clone https://github.com/ros-industrial-consortium/bezier.git
+mkdir catkin_workspace
+mkdir catkin_workspace/src
+cd catkin_workspace/src
+catkin_init_workspace
```

Comment:
This isn't needed. `catkin_make` (and `catkin build`) will do this automatically.